interface-definitions/system_option.xml.in

```diff
@@ -183,6 +183,26 @@
             </properties>
             <defaultValue>12-hour</defaultValue>
            </leafNode>
+           <leafNode name="usb-autosuspend">
+            <properties>
+              <help>USB autosuspend option</help>
+              <completionHelp>
+                <list>enable disable</list>
+              </completionHelp>
+              <valueHelp>
+                <format>enable</format>
```

Comment:
We agree to not use bool values like enable/disable, true/false, 1/0
use valueless option
If by default it is enabled, you can use
```
set system option disable-usb-autosuspend
```

Reply:
ok, fixed

Reply:
Does this mean that autosuspend will now be enabled by default (as it already were) unless `set system option disable-usb-autosuspend` is set?

Because files changed in this PR doesnt currently seem to reflect that behavior.

Reply:
yes, currently `autosuspend` is allowed by default. I made new changes after your comment.